extra/1-radio-stations.js

```diff
@@ -14,6 +14,16 @@
  */
 
 // `getAllFrequencies` goes here
+function getAllFrequencies(){
+  // const arrAllFrequencies =[87, 88, 89, 90, 91, 92, 93, 94, 95, 
+  //                       96, 97, 98, 99, 100, 101, 102, 103, 104,
+  //                       105, 106, 108, 108];
+  const arrAllFrequencies = [];
+  for (let i = 87; i< 109; i++){
```

Comment:
To avoid magic numbers here, it's worth having a variable for `startFrequency` and `endFrequency`

Reply:
That one was really challenging for me. I am going to try to understand what you've meant.

Reply:
The logic is done perfectly so you definitely don't need to worry about that 🙂, I just mean instead of having the numbers 87 and 109 in the function without explaining what they're for, it may be better to assign them to variables, so have 
```
const startFrequency = 87;
const endFrequency = 109;
```

Reply:
I got it now, thank you. I was studying on freecodecamp, I made smt like this in an exercise just now.